src/test/kotlin/lotto/services/LottoServiceTest.kt

```diff
@@ -0,0 +1,36 @@
+package lotto.services
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class LottoServiceTest {
+    @ValueSource(ints = [TICKET_PRICE, TICKET_PRICE * 2, TICKET_PRICE * 3])
+    @ParameterizedTest
+    fun `dose not throw if ticket price is valid`(ticketPrice: Int) {
+        assertDoesNotThrow {
+            LottoService.purchaseAmountValidator(ticketPrice)
+        }
+    }
+
+    @ValueSource(ints = [TICKET_PRICE - 1, TICKET_PRICE + 1, TICKET_PRICE * 2 + 1])
+    @ParameterizedTest
+    fun `throw if ticket price is invalid`(ticketPrice: Int) {
+        assertThrows<IllegalArgumentException> {
+            LottoService.purchaseAmountValidator(ticketPrice)
+        }
+    }
```

Comment:
Nice use of `@ValueSource` & `@ParameterizedTest` 👍 
Test names are also readable.
To go beyond, I recommend making test name conventions with your pair,
since having formatted, consistent test names really help the team maintain them,
especially when the numbers of tests go up.